adapters/webhdfs-adapter/src/test/java/com/connexta/replication/adapters/webhdfs/WebHdfsNodeAdapterFactoryTest.java

```diff
@@ -51,25 +51,7 @@ public void testCreate() {
   }
 
   @Test
-  public void testCreateNoTrailingSlash() {
-    NodeAdapter webHdfsNodeAdapter = webHdfsNodeAdapterFactory.create(urlWithPath);
-    assertThat(webHdfsNodeAdapter.getClass().isAssignableFrom(WebHdfsNodeAdapter.class), is(true));
-    assertThat(
-        ((WebHdfsNodeAdapter) webHdfsNodeAdapter).getWebHdfsUrl().toString(),
-        is(urlWithPath.toString()));
-  }
-
-  @Test
-  public void testCreateNoPath() throws MalformedURLException {
-    NodeAdapter webHdfsNodeAdapter = webHdfsNodeAdapterFactory.create(urlWithNoPath);
-    assertThat(webHdfsNodeAdapter.getClass().isAssignableFrom(WebHdfsNodeAdapter.class), is(true));
-    assertThat(
-        ((WebHdfsNodeAdapter) webHdfsNodeAdapter).getWebHdfsUrl().toString(),
-        is(urlWithNoPath.toString()));
-  }
```

Comment:
Removal of these tests will result in a loss of coverage of this block in the factory:
`    if (!baseUrl.endsWith("/")) {
      baseUrl = baseUrl.concat("/");
    }`

I would prefer that we keep them, but rather than use the class variables, create local variables within the test for these specific path cases.

Reply:
I misunderstood the intent of your comment. I've updated the tests.